src/commonMain/kotlin/character/classes/druid/talents/NaturesReach.kt

```diff
@@ -0,0 +1,19 @@
+package character.classes.druid.talents
+
+import character.Talent
+
+/**
+ *
+ */
+class NaturesReach(currentRank: Int) : Talent(currentRank) {
```

Comment:
The sim does not take any spell range into consideration, so this talent can probably be omitted.